src/parser/ast/ReturnStatmentNode.h

```diff
@@ -40,6 +40,10 @@ class ReturnStatmentNode : public StatementNode {
     virtual ASTNodeType type() { return ASTNodeType::ReturnStatement; }
     virtual void generateStatementByteCode(ByteCodeBlock* codeBlock, ByteCodeGenerateContext* context)
     {
+        if (codeBlock->m_codeBlock->isGenerator()) {
+            codeBlock->pushCode(GeneratorComplete(ByteCodeLOC(SIZE_MAX)), context, this);
+        }
+
         if (context->m_tryStatementScopeCount) {
```

Comment:
``if (context->m_tryStatementScopeCount != 0)`` or ``if (context->m_tryStatementScopeCount) > 0)`` matches it's meaning